src/App.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "./styles/colors" as *;
+
+.main {
+  position: relative;
+  display: flex;
+  height: 100vh;
+  flex-direction: column;
+}
+
+@media only screen and (min-width: 1024px) {
```

Comment:
in scss you could just nest this @media in first .main so there is no need for that duplication